src/stream/src/executor/sink.rs

```diff
@@ -621,6 +641,42 @@ impl<F: LogStoreFactory> SinkExecutor<F> {
                     if notify.send(()).is_err() {
                         warn!("failed to notify rebuild sink");
                     }
+                    log_reader.init().await?;
+                }
+                result = update_config_rx.recv()  => {
```

Comment:
shall we push a new sink instance to the `rebuild_sink_tx`, it basically do the same thing as vnode change.